extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -220,6 +289,76 @@ export class AmpSelector extends AMP.BaseElement {
   }
 
   /**
+   * Handles click events for the selectables.
+   * @param {!Event} event
+   */
+  clickHandler_(event) {
+    let el = dev().assertElement(event.target);
+    if (!el) {
+      return;
+    }
+    if (!el.hasAttribute('option')) {
+      el = closest(el, e => e.hasAttribute('option'), this.element);
+    }
+    if (el) {
+      this.onOptionSelected_(el);
+    }
+  }
+
+  /**
+   * Handles keyboard events for the selectables.
+   * @param {!Event} event
+   */
+  keyDownHandler_(event) {
+    // Make currently selected option unfocusable
+    this.options_[this.focusedIndex_].tabIndex = -1;
+
+    const isLtr = this.win.document.body.getAttribute('dir') != 'rtl';
+    let dir = 0;
+
+    switch (event.keyCode) {
+      case 37: // Left
+        // Left is considered 'previous' in LTR and 'next' in RTL.
+        dir = isLtr ? -1 : 1;
+        break;
+      case 38: // Up
+        // Up is considered 'previous' in both LTR and RTL.
+        dir = -1;
+        break;
+      case 39: // Right
+        // Right is considered 'next' in LTR and 'previous' in RTL.
+        dir = isLtr ? 1 : -1;
+        break;
+      case 40: // Down
+        // Down is considered 'next' in both LTR and RTL.
+        dir = 1;
+        break;
+    }
+
+    if (dir == 0) {
+      return;
+    }
+    event.preventDefault();
+
+    // Change the focus to the next element in the specified direction.
+    // The selection should loop around if the user attempts to go one
+    // past the beginning or end.
+    this.focusedIndex_ = (this.focusedIndex_ + dir) % this.options_.length;
+    if (this.focusedIndex_ < 0) {
+      this.focusedIndex_ = this.focusedIndex_ + this.options_.length;
+    }
+
+    // Focus newly selected option
+    const newSelectedOption = this.options_[this.focusedIndex_];
+    newSelectedOption.tabIndex = 0;
+    tryFocus(newSelectedOption);
+    if (!this.isMultiple_ &&
```

Comment:
We already assert this isn't multiple and `SELECT` above, so just check if this is `SELECT`.